behave_framework/src/minifi_test_framework/containers/container.py

```diff
@@ -418,3 +419,25 @@ def directory_contains_file_with_json_content(self, directory_path: str, expecte
                 continue
 
         return False
+
+    def directory_contains_file_with_minimum_size(self, directory_path: str, expected_size: int) -> bool:
+        if not self.container or not self.not_empty_dir_exists(directory_path):
+            return False
+
+        command = "sh -c {}".format(shlex.quote(f"find {directory_path} -maxdepth 1 -type f -exec stat -c %s {{}} \\;"))
```

Comment:
This should do it in one command, and the size filtering can be removed from below. Also I don't think wrapping in `sh -c` is necessary, but I've raised comment this in an earlier PR too
```suggestion
        command = f"find \"{directory_path}\" -maxdepth 1 -type f -size +{expected_size}"
```